paywall/src/data-iframe/cache/localStorage.js

```diff
@@ -126,7 +126,11 @@ export async function merge({
     delete container[type][subType]
   } else {
     container[type] = container[type] || {}
-    container[type][subType] = value
+    const oldType = container[type][subType] || {}
```

Comment:
we just chatted about this IRL but we guess this could be renamed to oldType

Reply:
just made the same mistake I did ;) renaming to `oldValue`